recipes/discosnp/meta.yaml

```diff
@@ -8,19 +8,22 @@ source:
   md5: "dfc35f6f5b83e7f661f5acbcb3c03bad"
 
 build:
-  number: 3
+  skip: True  # [not py3k]
+  number: 4
 
 requirements:
   build:
     - gcc   # [linux]
     - llvm  # [osx]
     - cmake
     - zlib {{CONDA_ZLIB}}*
+    - python
```

Comment:
I don't think `python` is needed as a build requirements, or am I wrong?

Reply:
You've right. There is a test during the install but the python requirement is not necessary.